modules/nw-ingress-controller-tls-profiles.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-tls-profiles_{context}"]
+= Ingress controller TLS profiles
+
+The `tlsSecurityProfile` parameter defines the schema for a TLS security profile. This object is used by operators to apply TLS security settings to operands.
+
+There are four TLS security profile types:
+
+* `Old`
+* `Intermediate`
+* `Modern`
+* `Custom`
+
+The `Old`, `Intermediate`, and `Modern` profiles are based on link:https://wiki.mozilla.org/Security/Server_Side_TLS#Recommended_configurations[recommended configurations]. The `Custom` profile provides the ability to specify individual TLS security profile parameters.
+
+.Sample `Old` profile configuration
+[source,yaml]
+----
+spec:
+  tlsSecurityProfile:
+    type: Old
+----
+
+.Sample `Intermediate` profile configuration
+[source,yaml]
+----
+spec:
+  tlsSecurityProfile:
+    type: Intermediate
+----
+
+.Sample `Modern` profile configuration
+[source,yaml]
+----
+spec:
+  tlsSecurityProfile:
+    type: Modern
+----
+
+The `Custom` profile is a user-defined TLS security profile.
+
+[WARNING]
+====
+You must be careful using a `Custom` profile as invalid configurations can cause problems.
```

Comment:
Per IBM, shouldn't use "as" to mean "because".

> Do not use to mean “because” or “while.”

Suggest also adding a comma:

```
 You must be careful using a `Custom` profile, because invalid configurations can cause problems.
```